frontend/src/features/admin-form/responses/IndividualResponsePage/IndividualResponsePage.tsx

```diff
@@ -186,34 +187,45 @@ export const IndividualResponsePage = (): JSX.Element => {
             </Text>
             {isPaymentLoading || isPaymentError ? (
               <Text>Payment was not enabled when this form was submitted</Text>
-            ) : (
+            ) : paymentData ? (
               <>
                 <Stack>
                   <Stack direction={{ base: 'column', md: 'row' }}>
                     <Text textStyle="subhead-1">Payment amount:</Text>
-                    <Text>{paymentData?.amount}</Text>
+                    <Text>
+                      S$
+                      {(paymentData.amount / 100).toLocaleString('en-IN', {
```

Comment:
Why `en-IN`? We should specify British English norms with `en-GB` ([link here](https://en.wikipedia.org/wiki/Language_localisation#Language_tags_and_codes)), or maybe `en-SG` (if that exists 😅)

```suggestion
                      {(paymentData.amount / 100).toLocaleString('en-GB', {
```

Reply:
changed here [759236d](https://github.com/opengovsg/FormSG/pull/5839/commits/759236d4a79610214035bdf0175e348cd4e11e3d)! (used `en-GB`)